scripts/firepit-builder/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "cloud_build",
+  "version": "1.0.0",
+  "description": "",
+  "main": "pipeline.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
`MIT` to match the repo.

And add `"private": true` since this should never be published.

Reply:
done & done brudda